src/ios/SimpleCameraPreview.m

```diff
@@ -47,8 +47,10 @@ - (void) enable:(CDVInvokedUrlCommand*)command {
     NSDictionary *setupSessionOptions;
     if (command.arguments.count > 0) {
         NSDictionary* config = command.arguments[0];
-        NSInteger targetSize = ((NSNumber*)config[@"targetSize"]).intValue;
-        setupSessionOptions = @{ @"targetSize" : [NSNumber numberWithInt:targetSize] };
+        if (config[@"targetSize"] != [NSNull null]) {
+            NSInteger targetSize = ((NSNumber*)config[@"targetSize"]).intValue;
```

Comment:
This will still crash if we have a value other than an integer here, right? To fix in later versions.

Reply:
@azharbeebeejaun We should check the type of integer also?

Reply:
Or just a try/catch.